pkg/common/vclib/storagepod_test.go

```diff
@@ -81,7 +81,12 @@ func TestDatastoreAndClusters(t *testing.T) {
 		objs = append(objs, stores[i].Reference())
 	}
 
-	_, err = pod.MoveInto(ctx, objs)
+	task, err := pod.MoveInto(ctx, objs)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	err = task.Wait(ctx)
```

Comment:
are we adding wait to prevent race conditions?

Reply:
The new simulator will run task async. We need to wait for the task to complete before we proceed with other operations